docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,182 @@
+==============
+Atop connector
+==============
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System and Process Monitor) Linux server performance analysis tool.
+
+Requirements
+------------
+
+In order to use this connector, the host on which the Trino worker is running
+needs to have the ``atop`` tool installed locally.
+
+Connector configuration
+-----------------------
+
+The connector can read disk utilization statistics on the Trino cluster.
+Create a catalog properties file that specifies the Atop connector by
+setting the ``connector.name`` to ``atop``.
+
+For example, create the file ``etc/catalog/system_monitor.properties``
+and replace the connector properties as appropriate for your setup:
+
+.. code-block:: text
+
+    connector.name=atop
+    atop.executable-path=/usr/bin/atop
+
+Configuration properties
+------------------------
+
+.. list-table:: Configuration variables
+  :widths: 40, 10, 10, 40
+  :header-rows: 1
+
+  * - Property name
+    - Default value
+    - Required
+    - Description
+  * - ``atop.concurrent-readers-per-node``
+    - ``1``
+    - Yes
+    - The number of concurrent read operations allowed per node.
+  * - ``atop.executable-path``
+    - (none)
+    - Yes
+    - The file path on the local file system for the ``atop`` utility.
+  * - ``atop.executable-read-timeout``
+    - ``1ms``
+    - Yes
+    - The timeout when reading from the atop process.
+  * - ``atop.max-history-days``
+    - ``30``
+    - Yes
+    - The maximum number of days in the past to take into account for statistics.
+  * - ``atop.security``
+    - ``ALLOW_ALL``
+    - Yes
+    - The :ref:`authorization checks<atop-authorization>` for the connector.
+  * - ``atop.time-zone``
+    - System default
+    - Yes
+    - The time zone identifier in which the atop data is collected. Generally the timezone of the host.
+      Sample time zone identifiers:
+
+      * ``Europe/Vienna``
+      * ``+0100``
+      * ``UTC``
+
+.. _atop-authorization:
+
+Authorization checks
+^^^^^^^^^^^^^^^^^^^^
+
+You can enable authorization checks for the connector by setting
+the ``atop.security`` property in the catalog properties file. This
+property must be one of the following values:
+
+.. list-table:: Atop security values
+  :widths: 30, 60
+  :header-rows: 1
+
+  * - Property value
+    - Description
+  * - ``ALLOW_ALL``
+    - No authorization checks are enforced.
+  * - ``SYSTEM``
+    - The connector relies on system-level access control.
```

Comment:
link to system access control https://trino.io/docs/current/security/built-in-system-access-control.html